aioapns/connection.py

```diff
@@ -413,12 +407,17 @@ async def send_notification(self, request):
                     "Could not send notification %s: " "ConnectionClosed",
                     request.notification_id,
                 )
+                raise
```

Comment:
Is it a mistake here? I don't think it's a good idea to raise an exception on `ConnectionClosed` because otherwise we won't try to send a notification `self.max_connection_attempts` times.

Reply:
It's not a mistake because I thought that ConnectionClosed usually means something exceptional like invalid certificate and in this case it's not worth it to try again.
But on the other hand this isn't the best decision to raise an exception because it seems like a breaking change for the library behaviour. So I'm going to remove this string now.